mypy/typeshed/stdlib/asyncio/unix_events.pyi

```diff
@@ -9,7 +9,7 @@ from .events import AbstractEventLoop, BaseDefaultEventLoopPolicy, _ProtocolFact
 from .selector_events import BaseSelectorEventLoop
 
 # This is also technically not available on Win,
-# but other parts of typeshed need this defintion.
+# but other parts of typeshed need this definition.
```

Comment:
The typeshed directory is vendored from https://github.com/python/typeshed. Could you remove typeshed changes from this PR and contribute them directly to the typeshed project instead?

Reply:
reverted and submitted upstream (some of the typos are already solved up there) https://github.com/python/typeshed/pull/7567